use_cluster.py

```diff
@@ -366,4 +355,391 @@ def return_cluster_dict(q_cluster, q_ids_list):
         else:
             final_clusters[q_cluster[indx]].append(q_ids_list[indx])
 
-    return final_clusters
\ No newline at end of file
+    return final_clusters
+
+
+"""
+> The objective of this function is to generate labels for each cluster based on a custom score developed by Araz, 
+using Normalised Mutual Information (NMI) and Distance from the Cluster Centroid 
+:The inputs required are the embeddings (generated previously from cluster functions), list of questions, question ids & clusters generated 
+: Read about NMI here: https://arxiv.org/pdf/1702.08199.pdf
+: For how Araz developed the custom score/metric, refer here: 
+"""
+
+def return_cluster_labels_NMI_nGrams_Centroid(embeddings,qs_list,q_ids_list,clusters): 
+
+    labelling_corpus = LabellingClusterCorpus(qs_list, nlp)
+    labelling_corpus.process_docs(embeddings, q_ids_list, clusters)
+    
+    
+    cluster_objs, vecs, global_keywords, lemmatized_qs_list = Generate_Modified_Qs_Data(labelling_corpus.documents, clusters, q_ids_list,1)
+    scores_1gram = Calculate_Label_Score(cluster_objs, vecs, global_keywords, lemmatized_qs_list)
```

Comment:
Just a note on code style: we typically use snake_case (with lowercase letters) for function names in Python rather than Capitalizing (which we resolve for classes, though without the underscores). You can review a standard style guide for Python here: https://www.python.org/dev/peps/pep-0008/

Reply:
Hey Jeff thanks for telling me this! :D I'll remember in the future! Have changed the function names accordingly 👍